tests/cursor.py

```diff
@@ -164,3 +196,111 @@ def test_uncallable_representation():
 def test_requests_volume_on_iteration(cursor):
     _ = [r for r in cursor]
     assert cursor.client.get.call_count == 1
+
+
+def test_skip_none(mocker):
+    cursor = mock_ordered_cursor(mocker, start=0, count=COLLECTION_SIZE)
```

Comment:
Why do you use this rather than your `ordered_cursor` fixture?

Reply:
No good reason in this case, updated